app/controllers/HomeController.scala

```diff
@@ -143,6 +143,12 @@ class HomeController @Inject()(val homeView: views.html.Home,
       case _ => Nil
     }
 
+  private def calculateOverduePaymentsCount(paymentsDue: List[LocalDate], outstandingChargesModel: List[OutstandingChargeModel]): Int = {
+    lazy val overduePaymentsCountFromDate = paymentsDue.count(_.isBefore(dateService.getCurrentDate))
```

Comment:
What's the benefit of using lazy values here if we're going to use them immediately?